### DISCUSSION TITLE
How to pause animation on last frame?

### DISCUSSION THREAD

**Discussion by saurabhbu** [google/model-viewer#3724] [Q&A]
I want to pause animation on last frame of animation, how can I do that?

**Comment by elalish** [comment]
Have you tried [`play({repetitions: 1})`](https://modelviewer.dev/docs/index.html#entrydocs-animation-methods-play)

**Reply by saurabhbu** [reply]
Yeah I tried, but when animation end the animation frame stop at first frame.

**Reply by elalish** [reply]
Hmm, that sounds like a bug. As a workaround, you could listen for the `finished` event and set currentTime to the end of the animation.

**Reply by saurabhbu** [reply]
got it. Thanks
